code/game/objects/items/weapons/storage/blackshield_kits.dm

```diff
@@ -0,0 +1,148 @@
+		new /obj/item/clothing/head/helmet/ballistic/militia(src)
+		new /obj/item/storage/pouch/ammo(src)
+
+// Primary Weapon Kits
+/obj/item/storage/box/bs_kit/mosin
+	name = "\improper Mosin Primary Kit"
+	desc = "The standard Blackshield equipment kit."
+
+	populate_contents()
+		new /obj/item/gun/projectile/boltgun(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/storage/pouch/ammo(src)
+
+/obj/item/storage/box/bs_kit/sts
+	name = "\improper STS Primary Kit"
+	desc = "The standard Blackshield equipment kit."
+
+	populate_contents()
+		new /obj/item/gun/projectile/automatic/sts/rifle/blackshield(src)
+		new /obj/item/ammo_magazine/rifle_75_short(src)
+		new /obj/item/ammo_magazine/rifle_75_short(src)
+		new /obj/item/ammo_magazine/rifle_75_short(src)
+		new /obj/item/ammo_magazine/rifle_75_short(src)
+		new /obj/item/ammo_magazine/rifle_75_short(src)
+		new /obj/item/storage/pouch/ammo(src)
+
+/obj/item/storage/box/bs_kit/cog
+	name = "\improper Cog Primary Kit"
+	desc = "A kit designed for energy-based weaponry."
+
+	populate_contents()
+		new /obj/item/gun/energy/retro(src)
+		new /obj/item/cell/medium(src)
+		new /obj/item/cell/medium(src)
+		new /obj/item/cell/medium(src)
+		new /obj/item/cell/medium(src)
+		new /obj/item/cell/medium(src)
+		new /obj/item/storage/pouch/ammo(src)
+
+/obj/item/storage/box/bs_kit/greasegun
+	name = "\improper Grease Gun Primary Kit"
+	desc = "The standard Blackshield equipment kit."
+
+	populate_contents()
+		new /obj/item/gun/projectile/automatic/greasegun(src)
+		new /obj/item/ammo_magazine/smg_35(src)
+		new /obj/item/ammo_magazine/smg_35(src)
+		new /obj/item/ammo_magazine/smg_35(src)
+		new /obj/item/ammo_magazine/smg_35(src)
+		new /obj/item/ammo_magazine/smg_35(src)
+		new /obj/item/storage/pouch/ammo(src)
+
+/obj/item/storage/box/bs_kit/grizzly
+	name = "\improper Grizzly Primary Kit"
+	desc = "The standard Blackshield equipment kit."
+
+	populate_contents()
+		new /obj/item/gun/projectile/shotgun/pump(src)
+		new /obj/item/ammo_casing/shotgun/prespawned(src)
+		new /obj/item/ammo_casing/shotgun/prespawned(src)
+		new /obj/item/ammo_casing/shotgun/prespawned(src)
+		new /obj/item/ammo_casing/shotgun/prespawned(src)
+		new /obj/item/ammo_casing/shotgun/prespawned(src)
+		new /obj/item/storage/pouch/ammo(src)
+
+// Side-arms Kits
+/obj/item/storage/box/bs_kit/stallion
+	name = "\improper Stallion Secondary Kit"
+	desc = "The standard Blackshield equipment kit."
+
+	populate_contents()
+		new /obj/item/gun/projectile/colt/ten/dark(src)
+		new /obj/item/ammo_magazine/magnum_40(src)
+		new /obj/item/ammo_magazine/magnum_40(src)
+		new /obj/item/clothing/accessory/holster/leg(src)
+
+/obj/item/storage/box/bs_kit/lamia
```

Comment:
Why would blackshield ***ever*** pick anything other then .50?

Reply:
Well I didn't know what else to put in the list.